packages/app-page-builder/src/types.ts

```diff
@@ -357,7 +368,11 @@ export type PbEditorPageElementPlugin = Plugin & {
         parent?: PbEditorElement
     ) => Omit<PbEditorElement, "id">;
     // A function to render an element in the editor.
-    render: (params: { theme?: PbTheme; element: PbEditorElement; isActive: boolean }) => ReactNode;
+    render?: (params: {
```

Comment:
There is one place in which `render` function is not provided. Can't tell if it's intended or not. Check `packages/app-page-builder/src/admin/utils/createElementPlugin.tsx` file - that's the one.